lib/views/pages/login_signup/register_form.dart

```diff
@@ -81,7 +78,9 @@ class RegisterFormState extends State<RegisterForm> {
       setState(() {
         _progressBarState = false;
       });
-      _exceptionToast(result.hasException.toString().substring(16, 35));
+      Fluttertoast.showToast(
+          msg: result.hasException.toString().substring(16, 35),
```

Comment:
What does the absolute values mean? can these be in a constant variable with descriptive names?

Reply:
For example resultsMessageStartPosition and resultsMessageEndPosition

Reply:
> What does the absolute values mean? can these be in a constant variable with descriptive names?

@dellyjm by absolute values do you mean the integer values (16 and 35) ?

Reply:
Already have a separate issue for error handling.

Reply:
> 
> 
> > What does the absolute values mean? can these be in a constant variable with descriptive names?
> 
> @dellyjm by absolute values do you mean the integer values (16 and 35) ?

yes, so name them so that the next developer reviewing understands what the start and end values mean without googling.

Reply:
@dellyjm yes it might be difficult to understand to the next developer.
So the developer who coded this has also added a[ print](https://github.com/PalisadoesFoundation/talawa/blob/f4e939d31b4b06bfb90ec423d63aa392007ea3e4/lib/views/pages/login_signup/register_form.dart#L80) statement to print the exception in the console.

Showing the whole exception to the user isn't a good idea.
So `result.hasException.toString().substring(16, 35)` displays only what the user may understand.
For eg:
This is printed in console (For devs):
`ClientException: Failed to connect to http://talawa-graphql-api.herokuapp.com/graphql/talawa/graphql: Failed host lookup: 'talawa-graphql-api.herokuapp.com'`

And a substring of that (from index 16 to 35) is for the user:
`Failed to connect`

These integer values may be different for different exceptions.
There is already a separate issue for error handling refer: issue #673